include/kernel/traps.h

```diff
@@ -20,6 +21,8 @@ static inline void c_entry_hook(void)
 #if defined(CONFIG_BENCHMARK_TRACK_KERNEL_ENTRIES) || defined(CONFIG_BENCHMARK_TRACK_UTILISATION)
     ksEnter = timestamp();
 #endif
+
+    debugLog(Entry);
```

Comment:
This approach of having `debugLog` be an empty `define` for builds not configured with the log still generates whitespace that triggers failure in the PR preprocess checks. I'm not sure whether it's the best approach.